webapp/src/js/bootstrapper/index.js

```diff
@@ -87,6 +87,17 @@
 
   /* pouch db set up function */
   module.exports = (POUCHDB_OPTIONS) => {
+    utils.checkApiAccessible()
```

Comment:
Another issue with this approach is if API becomes unavailable between this request and the next one then you'll still get the same infinite spinner.

The issue talks about the console getting loads of errors - can we catch those errors and update the UI based on them rather than creating a new request? That way there's no additional network traffic and it handles a range of errors and there's no race condition where this could still be an issue.